setup.py

```diff
@@ -17,7 +17,13 @@
     download_url="https://github.com/cognitedata/cognite-sdk-python/archive/{}.tar.gz".format(version),
     author="Erlend Vollset",
     author_email="erlend.vollset@cognite.com",
-    install_requires=["requests>=2.21.0,<3.0.0", "pandas", "requests_oauthlib==1.3.0"],
+    install_requires=[
+        "requests>=2.21.0,<3.0.0",
+        "pandas",
+        "requests_oauthlib==1.3.0",
+        "geopandas==0.10.0",
+        "shapely==1.7.0",
```

Comment:
Too strict. Let's allow patch bumps on 0-major versions and minor bumps and 1-major versions:
```suggestion
        "requests_oauthlib==1.*",
        "geopandas==0.10.*",
        "shapely==1.*",
```